index.js

```diff
@@ -95,6 +84,10 @@ for (const platform of platforms) {
                 platformConfig.features[key].exceptions = platformConfig.features[key].exceptions.concat(platformOverride.features[key].exceptions)
             }
         }
+        if (platformConfig.features[key].state !== 'enabled') {
```

Comment:
This doesn't seem future proof. Especially if we add features in beta/experimental state. I would think it would be better to check specifically for `disabled` if we're set on removing them from the config.

Reply:
None of the code caters for other states either, there's a much bigger project there.

I can make the change for id there's no state or "disabled" as mentioned though.